fetch-api/js/app.js

```diff
@@ -0,0 +1,70 @@
+const select = document.getElementById('breeds');
+const card = document.querySelector('.card');
+
+select.addEventListener('change', fetchBreedImage);
+card.addEventListener('click', fetchBreedImage);
+document.querySelector('form').addEventListener('submit', postInputFieldData);
+
+Promise.all([
+    fetchData('https://dog.ceo/api/breeds/image/random'),
+    fetchData('https://dog.ceo/api/breeds/list')
+])
+    .then(data => {
+        generateImage(data[0].message);
+        generateBreedOptions(data[1].message);
+    });
+
+//Functions:
+function checkStatus(response) {
+    if (response.ok) {
+        return Promise.resolve(response);
+    }
+    return Promise.reject(new Error(response.statusText));
+}
+
+function generateImage(data) {
```

Comment:
I would say `generateImageElement` to be even more precise.
At a quick glance I might think this is creating and/or saving an actual image file.